publish-github-release/index.js

```diff
@@ -38,8 +40,10 @@ async function setupCredentials(user, pass)
 
 async function run()
 {
+    const modulesJsonUrl = "https://raw.githubusercontent.com/VirtoCommerce/vc-modules/master/modules_v3.json";
     let branchName = await utils.getBranchName(github);
     let customModuleDownloadUrl = "";
+    let prereleasePackageUrl = "";
     if(branchName === 'dev')
     {
         let blobUrl = `https://vc3prerelease.blob.core.windows.net/packages${process.env.BLOB_SAS}`;
```

Comment:
blobUrl  should be defined as an input parameter for the action
with default value `https://vc3prerelease.blob.core.windows.net/packages`

Reply:
Fixed